build.sbt

```diff
@@ -35,11 +35,16 @@ libraryDependencies ++= Seq(
   "software.amazon.awssdk" % "dynamodb" % awsSdkVersionV2,
   "net.logstash.logback" % "logstash-logback-encoder" % "6.0",
   "com.gu" % "kinesis-logback-appender" % "1.4.4",
-  "com.github.nscala-time" %% "nscala-time" % "2.18.0",
-  "com.github.t3hnar" %% "scala-bcrypt" % "3.1",
+  "com.github.nscala-time" %% "nscala-time" % "2.30.0",
+  "com.github.t3hnar" %% "scala-bcrypt" % "4.3.0",
   "org.scanamo" %% "scanamo" % "1.0.0-M17",
-  "org.scalatest" %% "scalatest" % "3.0.5" % Test,
-  "com.gu" %% "anghammarad-client" % "1.1.3"
+  "org.scalatest" %% "scalatest" % "3.2.10" % Test,
+  "com.gu" %% "anghammarad-client" % "1.2.0"
+)
+
+dependencyOverrides ++= Seq(
+  "com.fasterxml.jackson.core" % "jackson-databind" % "2.11.4",
+  "com.github.blemale" %% "scaffeine" % "4.1.0"
```

Comment:
When we bump Java to 11, it should eliminate the need for this override.

Reply:
No harm in upgrading to `Java 11`, however, are you sure we need to? My understanding was this was only needed on Scala version `2.12.x`.

Reply:
I've removed this, and it now seems to work fine – can I double check what you're experiencing?

Reply:
Ah, I've now got it with an `sbt clean` 🤦  – I'll re-add this override, which should sort this issue.